router/core/graph_server.go

```diff
@@ -393,6 +397,18 @@ func (s *graphServer) buildGraphMux(ctx context.Context,
 		}
 	}
 
+	if s.securityConfiguration.MaxQueryDepth > 0 {
+		queryDepthCacheConfig := &ristretto.Config[uint64, bool]{
+			MaxCost:     1024,
```

Comment:
This currently is a magic number. Should I add the ability to configure the cache size to the config? If so, I'll do that there, similar to how the validation cache sizing works

Reply:
Please make it configurable, yes.
1024 as default sounds ok.